extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -393,10 +393,10 @@ function createDicomWebApi(dicomWebConfig, servicesManager) {
           // The value.Value will be set with the bulkdata read value
           // in which case it isn't necessary to re-read this.
           if (value && value.BulkDataURI && !value.Value) {
+            // handle the scenarios where bulkDataURI is relative path
+            fixBulkDataURI(value, naturalized, dicomWebConfig);
```

Comment:
This changes fixes pdf and video display for certain external URLs.  Probably should have been aprt of a different PR, so if you want me to move it, I can.